src/button/Button.tsx

```diff
@@ -62,7 +62,8 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
         autoFocus={shouldFocus}
         onClick={handleClick}
         disabled={isButtonDisabled}
-        {...rest}>
+        {...rest}
+      >
```

Comment:
This change reason is a new prettier rule? 🤔 Wasn't the previous format better? What do you think?

Reply:
I've checked it again, I think they renamed a rule (`bracketSameLine`), that was the reason, fixed it now, thanks!